doc/source/whatsnew/v0.25.0.rst

```diff
@@ -86,6 +86,7 @@ Other API Changes
 - :class:`DatetimeTZDtype` will now standardize pytz timezones to a common timezone instance (:issue:`24713`)
 - ``Timestamp`` and ``Timedelta`` scalars now implement the :meth:`to_numpy` method as aliases to :meth:`Timestamp.to_datetime64` and :meth:`Timedelta.to_timedelta64`, respectively. (:issue:`24653`)
 - :meth:`Timestamp.strptime` will now rise a ``NotImplementedError`` (:issue:`25016`)
+- Default `skipna=True` for :meth:`Series.pct_change` and :meth:`DataFrame.pct_change` will drop NAs before calculation (:issue:`25006`)
```

Comment:
this sounds like this is a change, but its actually not, you are just adding the ``skipna`` arg. pls make that more clear. put this in other enhancements.

Reply:
@jreback I am not _just_ adding `skipna` arg; I am setting `skipna=True` as default (before, the default was `fill_method='pad'`). I think this is an API breaking change. Indeed I quoted what you told me:

> the fill_method defaults to pad so not sure how this makes sense, aside from which this is an API change.

Reply:
right, maybe also say *adding* the ``skipna`` arg (as its not obvious that it was added in the note)

Reply:
can you say that thiis is current behavior and the default is NO change.